content/docs/faq.md

```diff
@@ -11,27 +11,64 @@ embed_js:
 
-### What is an "addon", exactly?
+### Who created Scratch Addons?
 
-An addon is similar to an extension or a userscript, but they use special APIs provided by the Scratch Addons extension. These APIs allow addons to run scripts on a Scratch page (userscripts), run scripts on the background (persistent scripts), or apply styles to the Scratch website (userstyles).
+Scratch Addons is a team project led by World_Languages. You can find the list of people who have contributed to us on the [Contributors](/contributors) page. While the "Developer tools" addon was initially created by griffpatch, he does not maintain the extension.
 
-Userscripts and persistent scripts can use the `addon.*` JavaScript APIs, which allow them to obtain Scratch-related information (for example, get the current logged in user) and also use extension APIs (like sending notifications).
+### What does Scratch Addons include?
 
-### If everything is an addon, then what does Scratch Addons do?
+Scratch Addons includes over 100 addons, which can be enabled or disabled individually. Some addons can also be customized on the Settings page. You can change each settings or use a preset to change color scheme or block styles. Scratch Addons also includes a popup, which can be used to enable addons quickly. Addons such as "Scratch messaging" provide additional functionalities to the popup. Scratch Addons is translated to many languages, including German, French, Spanish, and Japanese.
 
-By itself, Scratch Addons is just an addon loader. Its main tasks are:
+### Is this the same as TurboWarp Addons?
 
-- Allow users to enable, disable and configure addons.
-- Run addons and provide APIs to them.
-- Provide global state to addons (for example, the addon.auth API).
-- Pollute prototypes for use by addon userscripts.
-- Provide ways to access and modify Redux state.
-- Avoid addons from interfering with each other.
-- Avoid duplicate work from different addons.
+[TurboWarp](https://turbowarp.org/) has some addons from Scratch Addons which can be used on their editor without installing Scratch Addons. However, Scratch Addons also includes addons for the Scratch website and the popup display. Therefore, it is still useful to have Scratch Addons even if you only code using TurboWarp.
+
+## System Requirements
+
+### What are the system requirements for Scratch Addons?
+
+Scratch Addons is officially supported on [Google Chrome](https://google.com/chrome/) (version 80 and above) and [Mozilla Firefox](https://mozilla.org/firefox/) (version 74 and above). Scratch Addons should also work on other desktop browsers that are based on those browsers, such as:
+
+- [Microsoft Edge](https://microsoft.com/edge/) (version 80+)
+- [Opera](https://opera.com/download/) (version 67+)
+- [Vivaldi](https://vivaldi.com/download/) (version 2.11+)
+- [Brave](https://brave.com/download/) (version 1.3+)
+
+Additionally, Scratch Addons (for obvious reasons) requires Internet connection.
```

Comment:
TW desktop go brrr

Reply:
Yeah, it would be great to mention turbowarp desktop.